src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldExtendedLookupMultiTable.class.php

```diff
@@ -527,33 +526,33 @@ public function RemoveFieldIndex()
     {
         parent::RemoveFieldIndex();
 
-        $query = 'ALTER TABLE `'.MySqlLegacySupport::getInstance()->real_escape_string($this->sTableName).'` DROP INDEX `'.MySqlLegacySupport::getInstance()->real_escape_string($this->name).'_combined`';
-
-        MySqlLegacySupport::getInstance()->query($query);
-        $aQuery = array(new LogChangeDataModel($query));
-        TCMSLogChange::WriteTransaction($aQuery);
+        $this->dropIndexByName($this->name.self::TABLE_NAME_FIELD_SUFFIX);
+        $this->dropIndexByName($this->name.'_combined');
```

Comment:
Was this missing? I only see one drop so far?

Reply:
Yes. The index with the _table_name suffix was missing, which in combination with the new introduced related record deletion would lead to slow deletions everywhere. Now that i write this... i think we need a fix update, that ads this index. I will do so. :-(

Reply:
Added an update to add the missing index.